platform/default/mbgl/storage/offline_download.cpp

```diff
@@ -107,6 +107,7 @@ OfflineRegionStatus OfflineDownload::getStatus() const {
 
         case SourceType::Video:
         case SourceType::Annotations:
+        case SourceType::Image:
             break;
```

Comment:
Offline support added in [`3506bb5`](https://github.com/mapbox/mapbox-gl-native/pull/8968/commits/3506bb5cf752412095353336e44c0eee806fe6bb)